state/converters.go

```diff
@@ -14,13 +14,14 @@ import (
 	"github.com/ethereum/go-ethereum/core/types"
 )
 
-func convertToProcessBatchResponse(oldRoot common.Hash, txs []types.Transaction, response *pb.ProcessBatchResponse) (*ProcessBatchResponse, error) {
-	responses, err := convertToProcessTransactionResponse(oldRoot, txs, response.Responses)
+func convertToProcessBatchResponse(txs []types.Transaction, response *pb.ProcessBatchResponse) (*ProcessBatchResponse, error) {
+	responses, err := convertToProcessTransactionResponse(txs, response.Responses)
 	if err != nil {
 		return nil, err
 	}
 	return &ProcessBatchResponse{
 		CumulativeGasUsed:   response.CumulativeGasUsed,
+		DiscardedBatchByOOC: isBatchOutOfCounters(response.Responses),
```

Comment:
```suggestion
		UnprocesedBatch: isBatchProcessed(response.Responses),
```

In the future, we will face other situations that cause a batch is not processed, such as bad RLP encoding of the batch data. IMO it's better to keep this param more generic

Reply:
Agree